services/movie.py

```diff
@@ -1,11 +1,15 @@
+from typing import Optional
+
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+    genres_ids: Optional[list[int]] = None,
+    actors_ids: Optional[list[int]] = None,
+    title: Optional[str] = None,
```

Comment:
let's use | operator everywhere instead of Optional because it's an old approach

Reply:
Fix everywhere